src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -85,8 +85,13 @@ def __init__(self, feature_extractor: Union["SequenceFeatureExtractor", str], *a
             tokenizer ([`PreTrainedTokenizer`]):
                 The tokenizer that will be used by the pipeline to encode data for the model. This object inherits from
                 [`PreTrainedTokenizer`].
-            modelcard (`str` or [`ModelCard`], *optional*):
-                Model card attributed to the model for this pipeline.
+            chunk_length_ms (`int`, *optional*, defaults to 0):
+                The input length for in each chunk. If `0` then chunking is disabled (default). Only available for CTC
+                models.
+            stride_length_ms (`int`, *optional*, defaults to `chunk_length_ms / 6`):
```

Comment:
I would prefer here `stride_length_s` because:

- sampling rate is in the format samples/s 
- we mostly deal with seconds for other speech parameters: https://github.com/huggingface/transformers/blob/1bfa34770706298feb9a40e3fe02c39ffae1db51/examples/pytorch/speech-recognition/run_speech_recognition_ctc.py#L197
- I think the value should be of type `float` no matter what. It's just not an "integer" value for me logically and it might very well be that we have people that would like to have a stride of exactly 1024 samples. If this input would only be in integer it would make it difficult to do so. I would favor an input of float here and later throw an error if `stride_length_s * sampling_rate` is not an integer. Same for chunk_length_ms

Reply:
> we mostly deal with seconds for other speech parameters: 

Hammer argument, consistency wins it !

> I would favor an input of float here and later throw an error if `stride_length_s * sampling_rate` is not an integer. 

We cannot ever do that, there's no way floats can reliably fall onto integers. You have to purposefully round and cast to int to get something reliable. No biggy but still. Check the good old `0.1 + 0.2 = 0.300000004` in python console, similar issues arise all the time for integers.

> Same for chunk_length_ms